libraries/AP_Common/Location.cpp

```diff
@@ -282,10 +282,18 @@ ftype Location::get_distance(const Location &loc2) const
     return norm(dlat, dlng) * LOCATION_SCALING_FACTOR;
 }
 
-// return the altitude difference in meters taking into account alt
-// frame.  if loc2 is below this location then "distance" will be
-// positive.  ie. this method returns how far above loc2 this location
```

Comment:
some text has done missing

Reply:
I have incorporated the review comments.